### ISSUE TITLE
order by release_date

### ISSUE THREAD

**Issue by rareinator** [spotify/web-api#1002]
So trying to get the latest releases for a certain artists via the endpoint https://api.spotify.com/v1/artists/{id}/albums
The reason for this is so that i can send messages to users if a artist has releases new music
The issue im having is, that when setting include_groups=album,single
It dosent order by release_date. It first sends out every album ordered by release_date
And after that it sends out every single ordered by release_date.
So i tried not setting include_groups, but the exact same thing happens i saw the #359 issue, and there it was that that when not using include_groups it would order everything by release_date. But that seems to not be the case.
I would appreciate any help in the matter

**Comment by raffomania** [comment]
This is especially bad because it breaks the "official" fix for https://github.com/spotify/web-api/issues/127

**Comment by maracuja-juice** [comment]
I think you currently need to order by release_date yourself in the client.

**Comment by rareinator** [comment]
@maracuja-juice Yeah that's what I'm doing. But I would prefer to do it on the server. But i can't see any workarounds

**Comment by raffomania** [comment]
please spotify, my app is hitting the rate limit constantly because I have to fetch so many releases just to find out if there are new ones
